io/plugins/eu.esdihumboldt.hale.io.jdbc.msaccess/plugin.xml

```diff
@@ -11,6 +11,7 @@
       <driver
             class="net.ucanaccess.jdbc.UcanaccessDriver"
             id="eu.esdihumboldt.hale.io.jdbc.msaccess.driver"
+            isFileBased="true"
```

Comment:
I would be nice to do this configuration also for SpatiaLite, as it is also file based.